controllers/apps/components/stateless.go

```diff
@@ -209,6 +209,13 @@ func (c *statelessComponent) Status(reqCtx intctrlutil.RequestCtx, cli client.Cl
 	if c.runningWorkload == nil {
 		return nil
 	}
+
```

Comment:
Why dont we need  'updateComponentInfoToPods' here, and it seems that the componentBase.StatusWorklod does not handle with annotation patching.